packages/gui/src/components/notification/NotificationAnnouncementDialog.tsx

```diff
@@ -0,0 +1,87 @@
+import { Flex, Link, DialogActions, useOpenExternal, useOpenDialog, ConfirmDialog } from '@chia-network/core';
+import { Trans } from '@lingui/macro';
+import { Button, Dialog, DialogContent, DialogTitle, Typography } from '@mui/material';
+import React from 'react';
+
+export type NotificationAnnouncementDialogProps = {
+  notification: Notification;
+  onClose?: (value?: any) => void;
+};
+
+export default function NotificationAnnouncementDialog(props: NotificationAnnouncementDialogProps) {
+  const { notification, onClose } = props;
+
+  const openExternal = useOpenExternal();
+  const openDialog = useOpenDialog();
+
+  function handleClose() {
+    onClose?.(false);
+  }
+
+  const message = 'message' in notification ? notification.message : undefined;
+  const url = 'url' in notification ? notification.url : undefined;
+  const from = 'from' in notification ? notification.from : undefined;
+
+  async function handleURLClick() {
+    const canProcess = await openDialog(
+      <ConfirmDialog title={<Trans>Hang On</Trans>} confirmTitle={<Trans>Yes</Trans>} confirmColor="primary">
+        <Trans>This link will take you to {url}. Are you sure you want to go there?</Trans>
+      </ConfirmDialog>
+    );
+
+    if (canProcess) {
+      openExternal(url);
+    }
```

Comment:
Can we use the `useOpenUnsafeLink` hook for consistency? The dialog looks like this:

![image](https://github.com/Chia-Network/chia-blockchain-gui/assets/339312/33769a7e-2431-4e84-acc7-92ae7ec4653a)

Reply:
Thanks, I did not know that we have this already :)

Reply:
Updated